ftw/mobilenavigation/browser/resources/navigation.css

```diff
@@ -41,7 +41,7 @@ div.slideNavi.loading,
 #portal-globalnav.mobileNavigation {
```

Comment:
When I use this in selector with jQuery I can not find any elements on the page while mobilenavigation open nor closed.